client/src/components/pipelines/browser/Folder.js

```diff
@@ -894,6 +911,73 @@ export default class Folder extends localization.LocalizedReactComponent {
     await this.createPipeline({name, description, repository, token});
   };
 
+  createVersionedStorage = async ({
+    name,
+    description,
+    repository,
+    pipelineType
```

Comment:
@AleksandrGorodetskii since this method is used only for *Versioned Storages* creation, we don't need to pass `pipelineType` as an argument of the method - it will always be the same (`VERSIONED_STORAGE`).